src/codeu/chat/DerbyStore.java

```diff
@@ -269,7 +270,7 @@ public Message getMessage(Uuid messageid) throws SQLException, IOException {
 				// Creation of time object from database
 				Time time = Time.fromMs(creation);
 				
-				User user = new User(userid, name, time);
+				User user = new User(userid, name, password, time);
```

Comment:
Just thinking out loud, but I wonder why a `User` object needs to store its password? Note that in "real life" this would be a bit of a security risk, especially since you're storing the passwords as plain text.